src/utils/gui/globjects/GUIPolygon.cpp

```diff
@@ -150,11 +153,22 @@ TesselatedPolygon::drawTesselation(const PositionVector& shape) const {
             gluTessVertex(tobj, points + 3 * i, points + 3 * i);
         }
         gluTessEndContour(tobj);
-
+        size_t startIndex = shape.size() * 3;
+        for (size_t j = 0; j < myHoles.size(); j++) {
+            PositionVector hole = myHoles[j];
```

Comment:
Sorry for not mentioning it earlier but the same remark of iterating with a const ref applies here too.